src/routes/about/+page.svelte

```diff
@@ -12,13 +12,11 @@
 <div class="width_md prose">
 	<section class="panel prose">
 		<p>
-			Felt's mission is to make our lives better with technology. Our main product is <a
+			Felt's mission to make our lives better with technology. Our main product is <a
```

Comment:
```suggestion
			Felt's mission is to make our lives better with technology. Our main product is <a
```
I think we still need the `is` here